### ISSUE TITLE
A11y_Code.visualstudio.com_visual studio Learn page_Overview_Name Role Value :  Role is defined incorrectly as "Links" for the "Dropdowns".

### ISSUE THREAD

**Issue by Samhitha66** [microsoft/vscode-docs#5158]
## GitHub Tags:
#A11yMAS;#A11yTCS;#Win10;#WCAG4.1.2;#DesktopWeb;#A11ySev2;#ChromiumEdge;#P1-VisualStudioCodeServices-Web-Feb2022;#P1_Visual Studio Code Services;#Visual Studio Code Services;

## Environmental Details:
#URL: https://code.visualstudio.com/learn

## OS & Browser Details :
Microsoft Windows 10 Enterprise
Version 10.0.19044 Build 19044
Microsoft Edge
Version 99.0.1150.7 (Official build) dev (64-bit)

## Repro Steps:

1. Hit the URL: https://code.visualstudio.com/learn
2. Tab till any dropdown(Ex: Get started, student lessons, Collaboration, Educator Resources, Remote development)
3. Inspect the dropdown
4. Check whether the role is defined correctly or not.

## Note:
Same issue found in this URL : https://code.visualstudio.com/api

## Actual results:
Role is defined incorrectly as "Links" for the "Dropdowns".

## Expected results:
Role should be defined correctly as dropdown

## User Impact:
 Users will get confuse to differentiate  whether they are dropdowns or links.

## Attachments:

https://user-images.githubusercontent.com/99243079/153172042-c4cc7494-1702-49bf-8058-075173b37749.mp4

https://user-images.githubusercontent.com/99243079/153396976-4d9d8168-f726-4659-a653-963ec7dcadc3.mp4

**Comment by Samhitha66** [comment]
@gregvanl 
Checked again this bug in the below environment
For main issue URL: https://code.visualstudio.com/learn 
For note issue URL : https://code.visualstudio.com/api

For both issue the incorrect role is defined for dropdowns.
Please reopen this bug.
![Bug_5158](https://user-images.githubusercontent.com/99243079/157199760-d93aa90c-7149-4e40-bf89-8ddffb806b8a.png)
![Bug_5158_Note issue](https://user-images.githubusercontent.com/99243079/157199765-18fad5bd-0acf-4d0f-bd42-1c9cfb512725.png)

**Comment by rzhao271** [comment]
@Samhitha66 I do not see a dropdown role defined at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

**Comment by Samhitha66** [comment]
@rzhao271 These are combo boxes. So, the role should mention as Combo box -> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role

**Comment by Samhitha66** [comment]
@rzhao271 
As per the discussion with SME's 
The product team needs to remove the aria label and title attributes.
Please fix the issue and assign back to us .
If issue is not fixed please reopen this bug.
Thanks!

**Comment by rzhao271** [comment]
The title is there in case the text gets clipped. We're unable to conditionally add titles to the dropdown elements.
We can remove the aria labels.

**Comment by Samhitha66** [comment]
@gregvanl 
Checked again this bug in the below environment
For main issue URL: https://code.visualstudio.com/learn
For note issue URL : https://code.visualstudio.com/api

Even though it is defined as button the expanded and collapsed state is properly reading by the screen reader. 
So , the issue is fixed.

https://user-images.githubusercontent.com/99243079/158746041-9f7863ff-710b-470f-bfcf-80091169fa73.mp4


https://user-images.githubusercontent.com/99243079/158746345-0016c08f-ee72-4cda-b3d1-c08a17205962.mp4
